packages/discord.js/src/structures/User.js

```diff
@@ -346,6 +346,8 @@ class User extends Base {
    * Fetches this user's flags.
    * @param {boolean} [force=false] Whether to skip the cache check and request the API
    * @returns {Promise<UserFlagsBitField>}
+   * @deprecated <warn>This method is deprecated and will be removed in the next major version.
+   * Flags may still be retrieved via {@link User#fetch}.</warn>
    */
   fetchFlags(force = false) {
```

Comment:
Runtime deprecate as well?

Reply:
That would produce 2 deprecation warnings since this calls the method from the user manager where there is already a deprecation warning. I am not immediately sure how to make it "one or the other".

Reply:
Probably shared variable re the dep. warning? Just export it from util or something like that.

```js
let xOrYDeprecationWarningEmitted = false;

function emitXorYDeprecationWarning() {
  if (xOrYDeprecationWarningEmitted) return;

  xOrYDeprecationWarningEmitted = true;
  process.emitWarning('bleh');
}
```

Reply:
Should be done with 925398fb19a1d3415c19add7b7c515fd1636b807.